src/apps/loyalty/server/route_handlers.tsx

```diff
@@ -0,0 +1,5 @@
+import { NextFunction, Request, Response } from "express"
+
+export function Home(req: Request, res: Response, next: NextFunction) {
```

Comment:
might as well move all the route handlers to this file if we're gonna go that route